packages/hash/shared/src/types.ts

```diff
@@ -311,6 +325,7 @@ type DataTypeDefinition = TypeDefinition & { dataTypeId: VersionedUri };
 
 type LinkEntityTypeDefinition = TypeDefinition & {
   linkEntityTypeId: VersionedUri;
+  description?: string;
```

Comment:
Same as before for `EntityTypeCreatorParams`. Does `TypeDefinition` not contain `description`?

Reply:
True, addressed in https://github.com/hashintel/hash/pull/1489/commits/ae331c6d485b3adc957d258418d4f012f70ac308